src/componentLibrary/CoinPair/CoinPair.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Story,
+  Meta
+} from '@storybook/react';
+
+import { CoinPair, CoinPairProps } from './';
+
+const Template: Story<CoinPairProps> = args => <CoinPair {...args} />;
+
+const Small = Template.bind({});
+Small.args = {
```

Comment:
Since `Small` and `Large` are defaults, maybe rename to `DefaultSmall` and `DefaultLarge`

Reply:
This is a Storybook thing so that the values in the UI match the props :)

<img width="768" alt="Screenshot 2022-04-26 at 10 56 15" src="https://user-images.githubusercontent.com/40243778/165274621-34d12e7a-5b6b-41f2-8635-ee9cf1a27348.png">